src/storage/v2/inmemory/storage.cpp

```diff
@@ -139,6 +140,11 @@ InMemoryStorage::InMemoryStorage(Config config, StorageMode storage_mode)
     }
   }
 
+  if (config_.gc_python.type == Config::Gc::Type::PERIODIC) {
```

Comment:
What if Memgraph is used without query modules? Then it doesn't make sense to use resources for doing PERIODIC gc

Reply:
There are Python query modules that are there by default without MAGE and the timer is by default set to quite high (180 seconds) so it shouldn't affect perfomance noticeably. GC only affects the python modules and is run on separate thread (so it shouldn't affect the database at all)

Reply:
Separate thread still influences the machine's resources but understand what you mean.